_includes/doc-card.html

```diff
@@ -0,0 +1,24 @@
+<div class="col-12 col-md-3 mb-3 mb-md-3"{% if include.item.slug == "donate" %} id="donate"{% endif %}>
+    <a class="text-decoration-none" href="
+        {%- if include.item.external-link %}
+            {{- include.item.external-link }}
+        {%- else %}
+            {%- if site.lang != 'en' %}/{{ site.lang }}{% endif %}
+            {%- if include.item.internal-link %}
+                {{- include.item.internal-link }}
+            {%- else %}
+                {{- include.item.url | replace:'.html','' }}
+            {%- endif %}
+        {%- endif -%}
+        "{% if include.item.slug == "donate" %} data-toggle="modal" data-target="#donationModal"{% endif %}>
```

Comment:
How about moving the donation warning message to its own page? Then we don't need the special handling here. Things like this make it hard to maintain code because the caller has a hard time seeing what happens inside.

Reply:
From a UI perspective I prefer to keep it in a modal.

Reply:
On a page, we could show things like a button to open subscriptions with donation links, explain why it is important to support podcasters to keep the ecosystem open, etc

Reply:
Needs decision meeting comment: 

Add here:
```
<div class="col-12 col-md-3 mb-3 mb-md-3" id="card-{% include.item.slug %}">
```

On contribute page (above the popup layout):

```
$("#card-donate").attr("data-toggle", "modal").attr("data-target", "#donationModal")
```